test/ci/launch.yml

```diff
@@ -0,0 +1,50 @@
+---
+- hosts: localhost
+  connection: local
+  gather_facts: false
+  tasks:
+    - name: list available AMIs
+      ec2_ami_find:
+        region: '{{ aws_region }}'
+        ami_tags:
+          operating_system: '{{ operating_system }}'
+          image_stage: 'base'
+          ready: 'yes'
+        sort: 'creationDate'
+        sort_order: ascending
+        no_result_action: fail
+      register: ami_facts
+
+    - name: choose appropriate AMIs for use
+      set_fact:
+        origin_ci_aws_ami_id_candidate: '{{ item.ami_id }}'
+      with_items: '{{ ami_facts.results }}'
+      when: "'qe' not in item.tags"
+
+    - name: determine which AMI to use
+      set_fact:
+        aws_image: '{{ origin_ci_aws_ami_id_candidate }}'
+      when: aws_image is not defined
+
+    - include_tasks: create_aws_vm.yml
+      register: aws
+      with_items: "{{ vms }}"
+      loop_control:
+        loop_var: outer_item
+
+    - name: write the inventory
```

Comment:
I'm not going to block on this, but we need to get a dynamic inventory here eventually.

Reply:
Right, I'll rework `deprovision` to remove VMs by clusterid tag

Reply:
Done, deprovision now doesn't require inventory